app/views/concepts/_child_nodes.html.haml

```diff
@@ -1,7 +1,8 @@
-- output =""
-- for child in @children
-  - output << tree_link_to_concept(child: child, ontology_acronym: child.explore.ontology.acronym, active_style: '', node: @concept)
-  - if child.hasChildren
-    - output << tree_link_to_children(child: child, concept_schemes: @schemes)
-  - output << "</li>"
-= raw output
+= turbo_frame_tag child_id(@concept)+'_childs' do 
```

Comment:
think of how to remove this file because it's duplicated with draw tree helper

Reply:
Still to do;

Here the idea will be to create a component called for example `TreeChildren`